src/v/cluster/controller_backend.cc

```diff
@@ -373,6 +373,14 @@ ss::future<> controller_backend::reconcile_ntp(deltas_t& deltas) {
     bool stop = false;
     auto it = deltas.begin();
     while (!(stop || it == deltas.end())) {
+        if (has_non_replicable_op_type(*it)) {
+            /// This if statement has nothing to do with correctness and is only
```

Comment:
In the "copro handle non_replicable events" commit, it looks like we would vassert out if a del_non_replicable event came through, so this comment about the skip just being for logging looks incorrect